content/sections/pros.en.md

```diff
@@ -6,44 +6,33 @@ weight = 1
 
 type = "pros"
 
-using_another_system = "Using another system"
+using_another_system = "Using another system?"
 contact_us = "contact us"
 
 
 [[paragraphs]]
-title = "A ready-to-use platform"
+title = "Online Catalogue with checkout"
 content = """
-    We are offering you more than a simple online showcase. Choose between our attractive canvases specifically \
-    designed for optometrists, and we will help you integrate your branding as well as your products and services.
+The Internet has revolutionized the way people shop. Give customers the experience they are looking for and attract new customers along the way with your online catalogue.
+
     """
 
 [[paragraphs]]
-title = "Opfront integrates with your inventory management system"
+title = "Connected with your software"
```

Comment:
For someone without knowledge of the product, I don't think it means anything. I think we should be pretty straight forward on what it is.